CHANGELOG.md

```diff
@@ -7,6 +7,22 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+### Added
+
+- A `FallibleOutputPin` trait has been added. It is a version of the `OutputPin` trait
```

Comment:
I think this needs to be updated to match the feature based approach, and we probably need to document that this release is opt-in and the next will be opt-out.

Reply:
done